### ISSUE TITLE
Air_Quality entity is being depreciated 

### ISSUE THREAD

**Issue by ampersandru** [sarahhenkens/home-assistant-iocare#21]
Is there any concern or action needed for the breaking change under AirVisual in 2021.8.0?
https://www.home-assistant.io/blog/2021/08/04/release-20218/#breaking-changes
```
The Air Quality platform has been marked as deprecated, therefore the air_quality entity has been removed and replaced with sensor entities.

If you used the air_quality entity in your automations, scripts or Lovelace dashboards, you’ll need to update your configuration to match this change.
```

I noticed that my airmega uses the air_quality entity

**Comment by RobertD502** [comment]
Yes, the air quality entity will need to be rewritten as a sensor entity.

**Comment by RobertD502** [comment]
@ampersandru I have migrated the air_quality platform to the sensor platform. It is all part of the open PR, but I am not sure when all those changes will be merged.

In the meantime, you can add my repo to HACS `https://github.com/RobertD502/home-assistant-iocare` to install version 0.0.2 of this integration which features air_quality measurements as sensors along with a new timer service.
